docs/assets/hooks/footer_hooks.py

```diff
@@ -0,0 +1,3 @@
+from datetime import datetime
+def on_config(config, **kwargs):
+    config.copyright = f"Copyright © 2024 - {datetime.now().year} PyGreece"
```

Comment:
The `mkdocs` docs seem to indicate that we need to `return config` as well here. I've verified this works though, so maybe just add it for completeness.